pom.xml

```diff
@@ -930,6 +897,11 @@
                                     <artifactId>nullaway</artifactId>
                                     <version>${version.nullaway}</version>
                                 </path>
+                                <path>
+                                    <groupId>com.google.errorprone</groupId>
+                                    <artifactId>error_prone_annotations</artifactId>
+                                    <version>${version.error-prone}</version>
+                                </path>
```

Comment:
Just tested against `reactive-support`: it seems we can omit this. (Perhaps because I just merged #197, which does introduce an Error Prone/Guava dependency convergence error, but we'll fix that in #175.)